### ISSUE TITLE
New filter list "ABPVN" for Vietnamese users

### ISSUE THREAD

**Issue by snhv** [gorhill/uBlock#2170]
Hi, 

Please include ABPVN, a Vietnamese filter list (the current Vietnamese list by Fanboy is so outdated and ineffective).
Filter list homepage: http://abpvn.com/get
Filter list link: abp:subscribe?location=http://abpvn.com/filter/abpvn-F3fkGO.txt&title=ABPVN%20List

Please also put Steven Black's hosts into 3-rd party pane.

Thank you

**Comment by Czech123** [comment]
Unique id in link maybe be a problem.
> Please keep carefully this link in a safe place and not shared on the forum or shared with others.

**Comment by snhv** [comment]
Yep, but Adguard has it, I'm not sure how though.

**Comment by toshiya44** [comment]
These filters are sure to create lots of false positives:
```_ads_
/qc/*
/adv/*
/ad/*
```
maybe these too
```/120x240.

/120x60.

/468-x-60-

/600x60_

_690x110.

_800x105.

_990x100.
```
There's this odd filter `29847` at the bottom, what is it?

I'm not sure if this filter is valid
`@@#-ad-util-/ad-iframe-/ads/160./ads/footer_/adsense5./rcolads1./sponsored_text./tmnadsense-ad=advertorial&/728x90_`

**Comment by snhv** [comment]
False positives: probably yes, but I've been using this filter for almost half a year now with no problem. For other issues, as I'm neither the maintainer nor technical, I'm unable to answer you, though you may reach out to the maintainer himself on his Facebook page (he's very responsive).

Anyway, this filter will be used only by its intended audience (Vietnamese) and I don't see others complaint.

**Comment by joey04** [comment]
@gorhill FYI can be closed since you recently added it.

**Comment by gorhill** [comment]
Fixed with 6ec3412b.
